.github/workflows/validate-renovate-config.yaml

```diff
@@ -0,0 +1,20 @@
+name: "Validate Renovate Configuration"
+
+on:
+  push:
+    paths:
+    - "renovate-config.json"
+  pull_request:
+    paths:
+    - "renovate-config.json"
+
+jobs:
+  validate:
+    # Only run on PRs if the source branch is on someone else's repo
+    if: "${{ github.event_name != 'pull_request' || github.repository != github.event.pull_request.head.repo.full_name }}"
```

Comment:
Why don't we validate our own changes? I don't quite understand what this is about

Reply:
Yeah, when a pull request is made from a different branch on the same repository, rather than a fork, both push and pull_request events trigger -- this skips one of them to prevent duplicate runs:

![image](https://user-images.githubusercontent.com/629092/158463796-a1b9cb56-0a99-4f47-ad03-69f5eed0bc55.png)